docs/immortal-empires/kislev.md

```diff
@@ -0,0 +1,32 @@
+# Kislev
+
+## Kostaltyn: The Great Orthodoxy
+
+* **Route I - There Can Only Be One**
+  * PLACEHOLDER:
+    * Erengrad, Praag, Kislev
+  * Have at least 600 Supporters
+  * **Reward:**
+    * _PLACEHOLDER_: Ice Court training cost + duration -X
```

Comment:
Turn this reward into +4 Public Order and +60 Growth. Maybe something else but not super strong.

Reply:
Perhaps just give both public order + growth, and the ice court reductions? Rewards related to faction mechanics are worth keeping I think?

Reply:
They are worth, but they do not suit this particular lord. Ice Court buffs should be for Katarin Bokha. Look for Motherland/Ataman effects.

Reply:
Good point, will remove and look around.

Reply:
Motherland:
wh3_main_effect_ability_enable_blood_of_the_motherland
wh3_main_effect_ksl_motherland_cost_mod
wh3_main_effect_ksl_motherland_cost_mod_hidden
wh3_main_effect_province_growth_motherland
wh3_main_effect_public_order_invoking_the_motherland

Ataman:
wh3_main_effect_corruption_reduction_ataman
wh3_main_effect_province_growth_ataman
wh3_main_effect_public_order_atamans

I have no idea what these are, I don't and haven't played Kislev

Reply:
The Ataman ones are not good for what we are looking for. I think **wh3_main_effect_ksl_motherland_cost_mod** is the one we want to taget here.

Reply:
Adding motherland_cost_mod and testing